stm32f0_hal/src/uart.rs

```diff
@@ -131,11 +131,11 @@ impl Uart {
                             .disabled()
                     });
                     // Configure UART : baudrate
-                    uart1.brr.write(|w| {
+                    uart1.brr.modify(|_, w| {
```

Comment:
I think those ones (baudrates registers) shouldn't be modify but write, @csagonero ?

Reply:
Must be write ...